pdns/dynhandler.cc

```diff
@@ -269,8 +269,8 @@ string DLNotifyHandler(const vector<string>&parts, Utility::pid_t ppid)
   UeberBackend B;
   if(parts.size()!=2)
     return "syntax: notify domain";
-  if(!::arg().mustDo("master"))
-      return "PowerDNS not configured as master";
+  if(!::arg().mustDo("master") && !::arg().mustDo("slave-renotify"))
+      return "PowerDNS not configured as master or slave with re-notifications";
```

Comment:
We only notify 'all' if kind = 0 (master) so slave domains never get notified, and yes behaviour of #3663 is similar.
And second, if slave=no should we send out slave renotifies when notifying all?